### DISCUSSION TITLE
How to resize mx.NDArray 2d tensor with interp = 'Nearest'

### DISCUSSION THREAD

**Discussion by PKurnikov** [apache/incubator-mxnet#10674] [Q&A]
I need function like mx.nd.SpatialTransformer or BilinearResize2D, but interp parametr is {'bilinear'}, required.

**Comment by lilhope** [comment]
Using the mx.nd.Upsampling

**Comment by PKurnikov** [comment]
@lilhope, Sounds tempting, but the function does not allow downscalling, because C-function parametr "scale" has unsigned type. 
`upsampling_nearest(const Exp<SrcExp, DType, etype> &src, index_t scale) `
It seems that it is necessary to rewrite С-structures to get required result.

**Comment by nswamy** [comment]
Please use https://discuss.mxnet.io for questions, its the best place to get answers/feedback from maintainers and other users. I will be closing this issue.

**Comment by nswamy** [comment]
Actually this looks a feature-request, I'll reopen this so we can prioritize this. You might want to still ask question on discuss to see if there are workarounds.
